django-stubs/forms/fields.pyi

```diff
@@ -125,8 +125,8 @@ class DecimalField(IntegerField):
     def __init__(
         self,
         *,
-        max_value: Optional[int] = ...,
-        min_value: Optional[int] = ...,
+        max_value: Optional[Decimal] = ...,
```

Comment:
```python
>>> from decimal import Decimal
>>> Decimal(1.5) > 1.2
True
>>> Decimal(1.5) > 1
True
```

Let's include `int` and `float`

Reply:
Alright, I've included them now. Kinda disagree about floats though, because the imprecise nature of floats. If you've decided to use a decimal field I assume you want accuracy, even though a float technically works.